Core/Core/Login/LoginUsePolicy/LoginUsePolicyView.swift

```diff
@@ -0,0 +1,96 @@
+
+    var body: some View {
+        VStack(spacing: 0) {
+            Text("Either you're a new user or the Acceptable Use Policy has changed since you last agreed to it. Please agree to the Acceptable Use Policy before you continue.", bundle: .core)
+                .font(.regular16).foregroundColor(.textDarkest)
+                .padding(EdgeInsets(top: 24, leading: 16, bottom: 16, trailing: 16))
+            Divider().padding(.zero)
+            Button {
+                env.router.route(to: "/accounts/self/terms_of_service", from: controller)
+            } label: {
+                HStack {
+                    Text("Acceptable Use Policy", bundle: .core)
+                        .font(.semibold16).foregroundColor(.textDarkest)
+                    Spacer()
+                    InstDisclosureIndicator()
+                }
+                .padding()
+                .frame(height: 47)
+                .contentShape(Rectangle())
+            }
+            .buttonStyle(ContextButton(contextColor: Brand.shared.primary))
+            Divider().padding(.zero)
+            HStack {
+                Toggle(isOn: $viewModel.isAccepted) {
+                    Text("I agree to the Acceptable Use Policy.", bundle: .core)
+                        .font(.semibold16).foregroundColor(.textDarkest)
+                        .fixedSize()
+                        .lineLimit(1)
+                }
+            }
+            .padding()
+            .frame(height: 47)
+            Divider()
+            Spacer()
+        }
+        .background(Color.backgroundLightest.edgesIgnoringSafeArea(.all))
+        .navigationBarStyle(.color(Brand.shared.primary))
+        .navigationTitle(Text("Acceptable Use Policy", bundle: .core))
+        .navigationBarItems(
+            leading: Button(action: {
+                controller.value.dismiss(animated: true) {
+                    viewModel.cancelAcceptance()
+                }
+            }, label: {
+                Text("Cancel", bundle: .core)
+                    .font(.regular16)
+                    .foregroundColor(.textLightest)
+            }),
+            trailing: Button(action: {
+                controller.value.dismiss(animated: true) {
+                    viewModel.submitAcceptance()
+                }
+            }, label: {
+                Text("Submit", bundle: .core)
+                    .font(.semibold16)
+                    .foregroundColor(.textLightest)
+                    .opacity(viewModel.isAccepted ? 1 : 0.4)
+            }).disabled(!viewModel.isAccepted)
+        )
+        .alert(isPresented: $viewModel.showError) {
+            Alert(title: Text(viewModel.errorText ?? NSLocalizedString("Something went wrong", comment: "")))
+        }
+    }
+}
+
+struct LoginUsePolicyView_Previews: PreviewProvider {
+    static var previews: some View {
+        let viewModel = LoginUsePolicyViewModel(accepted: {}, cancelled: {})
+        LoginUsePolicyView(viewModel: viewModel)
+    }
+}
```

Comment:
To be consistent with previews we should move this into an '#if DEBUG' statement. The factory logic could be also moved to the assembly file with the same #if DEBUG statements.